lib/graphql/client.ex

```diff
@@ -107,4 +107,34 @@ defmodule BridgeEx.Graphql.Client do
       log_response_on_error: false
     )
   end
+
+  @doc """
+  Formats Graphql query variables to make it compliant with the Schema
+  """
+  @spec format_variables(any) :: any
+  def format_variables(nil), do: nil
+  def format_variables(variable = %Date{}), do: Date.to_string(variable)
+  def format_variables(variable = %DateTime{}), do: DateTime.to_string(variable)
+  def format_variables(variable = %NaiveDateTime{}), do: NaiveDateTime.to_string(variable)
```

Comment:
I think it is better doing this at application level. Not all applications want to convert dates in this way, for instance a `DateTime` could be converted to Epoch or ISO8601.